src/components/app-card/AppCard.jsx

```diff
@@ -2,9 +2,9 @@ import Box from '@mui/material/Box'
 
 import { styles } from '~/components/app-card/AppCard.styles'
 
-const AppCard = ({ children, isClickable = true }) => {
+const AppCard = ({ children, isClickable = true, containerCardStyles }) => {
```

Comment:
it`s a component, use default naming for props not containerCardStyles etc

Reply:
fixedd